ts/screens/main/MessagesScreen.tsx

```diff
@@ -67,26 +75,58 @@ class MessagesScreen extends React.Component<Props, never> {
     ) : null;
   };
 
-  private renderMockedMessages = (
+  public renderItem = (messageDetails: IMessageDetails) => {
+    return (
+      <MessageComponent
+        key={messageDetails.item.id}
+        date={messageDetails.item.created_at}
+        services={this.props.services}
```

Comment:
why pass all services to a component that displays a single message? you should pass only the relevant service